docs/operator-manual/user-management/microsoft.md

```diff
@@ -92,107 +80,129 @@
       - **Redirect URI:** `http://localhost:8085/auth/callback`
       ![Azure App registration's Authentication](../../assets/azure-app-registration-authentication.png "Azure App registration's Authentication")
 
-#### Add credentials a new Azure AD App registration
+##### Add credentials a new Azure AD App registration
 
 1. From the `Certificates & secrets` menu, choose `+ New client secret`
 2. Enter a `Name` for the secret (e.g. `ArgoCD-SSO`).
       - Make sure to copy and save generated value. This is a value for the `client_secret`.
       ![Azure App registration's Secret](../../assets/azure-app-registration-secret.png "Azure App registration's Secret")
 
-#### Setup permissions for Azure AD Application
+##### Setup permissions for Azure AD Application
 
 1. From the `API permissions` menu, choose `+ Add a permission`
 2. Find `User.Read` permission (under `Microsoft Graph`) and grant it to the created application:
    ![Azure AD API permissions](../../assets/azure-api-permissions.png "Azure AD API permissions")
 3. From the `Token Configuration` menu, choose `+ Add groups claim`
    ![Azure AD token configuration](../../assets/azure-token-configuration.png "Azure AD token configuration")
 
-### Associate an Azure AD group to your Azure AD App registration
+#### Associate an Azure AD group to your Azure AD App registration
 
 1. From the `Azure Active Directory` > `Enterprise applications` menu, search the App that you created (e.g. `Argo CD`).
       - An Enterprise application with the same name of the Azure AD App registration is created when you add a new Azure AD App registration.
 2. From the `Users and groups` menu of the app, add any users or groups requiring access to the service.
    ![Azure Enterprise SAML Users](../../assets/azure-enterprise-users.png "Azure Enterprise SAML Users")
 
-### Configure Argo to use the new Azure AD App registration
-
-1. Edit `argocd-cm` and configure the `data.oidc.config` and `data.url` section:
+#### Configure Argo to use the new Azure AD App registration
 
-            ConfigMap -> argocd-cm
+1. Edit `argocd-cm` ConfigMap and configure the `data.oidc.config` and `data.url` section:
 
-            data:
-               url: https://argocd.example.com/ # Replace with the external base URL of your Argo CD
-               oidc.config: |
-                     name: Azure
-                     issuer: https://login.microsoftonline.com/{directory_tenant_id}/v2.0
-                     clientID: {azure_ad_application_client_id}
-                     clientSecret: $oidc.azure.clientSecret
-                     requestedIDTokenClaims:
-                        groups:
-                           essential: true
-                     requestedScopes:
-                        - openid
-                        - profile
-                        - email
+```yaml
```

Comment:
Unfortunately, the absurd-looking indentation is necessary to maintain list ordering when rendered by mkdocs. You can test with `make server-docs-local1.

<img width="720" alt="image" src="https://user-images.githubusercontent.com/350466/183089177-b307b11e-c63c-4db0-96b2-3e04e2a9c323.png">

Reply:
Thanks @crenshaw-dev ! I looked through the docs with `make serve-docs-local` but obviously missed that indention. Should be fixed now. Let me know if you have any remarks or need further examples.